### DISCUSSION TITLE
I want tabs and accordion elements

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#162] [Wants]
Stop making people poorly and inaccessibly re-implement common UI controls we&#39;ve been using for decades. Give us more accessible controls.

<hr>

https://webwewant.fyi/wants/21/

**Comment by mincerafter42** [comment]
Now achievable with the new `name` attribute on the `<details>` element I think.

**Reply by aarongustafson** [reply]
Accordion is, yes.
